recipes/metameta/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: metameta
+  version: 1.0
+
+source:
+  fn: v1.0.tar.gz
+  url: https://github.com/pirovc/metameta/archive/v1.0.tar.gz
+  md5: 0214502abbb8b6e9c4d49b74f2e1d58f
+
+build:
+  number: 0
+  skip: True # [osx]  
+  
+requirements:
+  run:
+    - snakemake >=3.9.0
+
+test:
+  commands:
+    - touch "metameta_dummy_file_test"
+    - metameta --configfile $PREFIX/opt/metameta/scripts/metameta_dummy_file_test.yaml -np > /dev/null
```

Comment:
Please remove the piping into /dev/null (harder to debug if something goes wrong).